pkgs/top-level/python-packages.nix

```diff
@@ -5281,11 +5281,11 @@ in {
 
   pytest-localserver = buildPythonPackage rec {
     name = "pytest-localserver-${version}";
```

Comment:
Can you move this expression to `pkgs/development/python-modules/pytest-localserver/default.nix`. See other files there on what to do with `pname`.